samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -1101,8 +1101,10 @@ int main(void)
 		}
 		printk("Streams started\n");
 
-		/* Start send timer */
-		k_work_schedule(&audio_send_work, K_MSEC(0));
+		if (CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT) {
+			/* Start send timer */
+			k_work_schedule(&audio_send_work, K_MSEC(0));
+		}
```

Comment:
This does not seem related to the overlay, and probably should be a different commit

```suggestion
		if (IS_ENABLED(CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT)) {
			/* Start send timer */
			k_work_schedule(&audio_send_work, K_MSEC(0));
		}
```

Reply:
Oops, I set `CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT=2` and was wondering why sending suddenly stopped working, until I read that: IS_ENABLED(config_macro) returns 1 if config_macro is defined to 1, 0 otherwise (including if config_macro is not defined). Nice prank :)

Reply:
Maybe there is some alternative to IS_ENABLED? I can't find it.